src/binder_plugin.c

```diff
@@ -681,20 +709,54 @@ binder_plugin_slot_data_role_changed(
     }
 }
 
+static
+RADIO_AIDL_INTERFACE
+binder_plugin_modem_interface(
+    BinderSlot* slot)
+{
+    if (slot->plugin->settings.interface_type == RADIO_INTERFACE_TYPE_AIDL) {
```

Comment:
Nitpick: kind of unnecessary `{}` in many cases but me, does not harm really. Would just look cleaner.

Reply:
Code style used everywhere in this project always uses {} even for single line ifs.